app/controllers/participants/index.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    // Manually specify column format options for model table
+    columns: [
+        {propertyName: 'id', title:'ID'},
+        {propertyName: 'firstName'},
+        {propertyName: 'lastName'},
+        {propertyName: 'modifiedOn', title: 'Last active'},  // TODO: That's not quite what this field does, but useful first approximation
+        {title: '', template: 'custom/profile-link'}
```

Comment:
Would you be willing to update this in 17 to something like: `table-templates` or similar?